frontend/public/components/routes/create-route.tsx

```diff
@@ -181,45 +181,49 @@ class CreateRouteWithTranslation extends React.Component<
   };
 
   addAltServiceEntry = () => {
-    this.setState(({ alternateServices }) => {
+    this.setState(({ alternateBackends }) => {
       const services = [
-        ...alternateServices,
+        ...alternateBackends,
         { name: null, weight: 100, key: _.uniqueId('alternate-backend-') },
       ];
-      this.props.formik.setFieldValue('formData.alternateServices', services);
+      this.props.formik.setFieldValue('formData.alternateBackends', services);
       return {
-        alternateServices: services,
+        alternateBackends: services,
       };
     });
   };
 
   removeAltServiceEntry(alternateServiceIndex: number) {
-    this.setState(({ alternateServices }) => {
-      const updatedServiceEntriesArray: AlternateServiceEntryType[] = [...alternateServices];
-      updatedServiceEntriesArray.splice(alternateServiceIndex, 1);
-      this.props.formik.setFieldValue('formData.alternateServices', updatedServiceEntriesArray);
-      if (updatedServiceEntriesArray.length < 1) {
-        this.setState({ weight: 100 });
-        this.props.formik.setFieldValue('formData.weight', 100);
-      }
-      return {
-        alternateServices: updatedServiceEntriesArray,
-      };
-    });
+    this.setState(
```

Comment:
Fixes a console error resulting from the nested `setState` calls.  Fix is to move the second `setState` to a callback.